### PR TITLE
fix test cases that may be incorrect

### PR DESCRIPTION
**Pull request by yangjunmyfm192085**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
A different ipv6 address was used in the wrong test cases. Might not be able to achieve what needs to be tested
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
 "NONE"
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/util/ipvs/ipvs_test.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func TestVirtualServerEqual(t *testing.T) {
 				Timeout:   0,
 			},
 			svcB: &VirtualServer{
-				Address:   netutils.ParseIPSloppy("2012::beeef"),
+				Address:   netutils.ParseIPSloppy("2012::beef"),
```
Comment (aojea) [inline pkg/util/ipvs/ipvs_test.go:82]: good catch 	fmt.Println(net.ParseIP("2012::beeef"))

**Comment by danwinship** [comment]
/assign @thockin 
should we add `sig-network-approvers` to `pkg/util/ipvs/OWNERS` and `pkg/util/iptables/OWNERS`?

**Comment by thockin** [comment]
@danwinship yes

/approve

**Pull request merged**